lib/libmbedtls/mbedtls/library/cmac.c

```diff
@@ -199,11 +198,26 @@ static void cmac_pad( unsigned char padded_block[MBEDTLS_CIPHER_BLKSIZE_MAX],
     }
 }
 
+int mbedtls_cipher_cmac_setup(mbedtls_cipher_context_t *ctx)
+{
+    mbedtls_cmac_context_t *cmac_ctx;
+
+    /* Allocated and initialise in the cipher context memory for the CMAC
+     * context */
```

Comment:
s/Allocated/Allocate/

Maybe use the std style:
```
   /*
    * Allocate and initialise a CMAC context memory in the cipher
    * context memory.
    */
```

Reply:
OK, it will be better. I will update it.